azure-mgmt-containerregistry/src/main/java/com/azure/management/containerregistry/implementation/RegistryImpl.java

```diff
@@ -94,9 +92,9 @@ public RegistryImpl update() {
     }
 
     @Override
-    public Completable afterPostRunAsync(boolean isGroupFaulted) {
+    public Mono<Void> afterPostRunAsync(boolean isGroupFaulted) {
         this.webhooks.clear();
```

Comment:
Is this sync or async?

Reply:
After my check, it is a collections clear. Only clear the local map.